app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<%= link_to 'Home', root_path, class: "" %>
+
+<%= link_to 'Events', events_path %>
+<% if logged_in? %>
+<%= link_to 'Create Event', new_event_path, class: "" %>
+<% end  %>
+<%= link_to 'Users', users_path %>
+
+
+    
+    <% if logged_in? %>
+     <%= link_to('My Profile', user_path(session[:user_id])) %>
+     <%= link_to('Sign out', logout_path, method: :delete) %>
+    <% else  %>
+    <%=  link_to('Log in', login_path) %>
+    <%=  link_to('Sign up', login_path,) %>
```

Comment:
I couldn't signup to the App because there is no signup_path. We accidentally used the `login_path` for the `signup_path`. Let us fix this

Reply:
Fixed